scalameta/scalameta/src/test/scala/scala/meta/tests/prettyprinters/SyntacticSuite.scala

```diff
@@ -184,6 +184,27 @@ class SyntacticSuite extends scala.meta.tests.parsers.ParseSuite {
     assert(Dotty(q"trait T(a: Int)").syntax === "trait T(a: Int)")
   }
 
+  test("literalTypes") {
+    intercept[ParseException] {
+      dialects.Scala211("val a : 42 = 42").parse[Stat].get.show[Syntax]
+    }
+    val Scala211 = null // TODO: #389
+    import scala.meta.dialects.Dotty
+    assert(q"val a: 42 = 42".show[Syntax] === "val a: 42 = 42")
```

Comment:
This is not supported yet either by Dotty or Typelevel Scala...

If inputting this expressing, a Parser Exception is going to be thrown, because neither the `+` and the `*` operators are implemented.
<img width="722" alt="captura de pantalla 2017-02-06 a las 20 33 58" src="https://cloud.githubusercontent.com/assets/10242025/22663315/71fee014-ecac-11e6-965a-ff4cfe0ad505.png">
<img width="695" alt="captura de pantalla 2017-02-06 a las 20 39 33" src="https://cloud.githubusercontent.com/assets/10242025/22663323/758347ac-ecac-11e6-9865-5e8ddfdf4daf.png">